templates/user/settings.html

```diff
@@ -402,14 +402,14 @@ <h3 class="h5">Update Email</h3>
                     <form id="email-form" action="settings/email" method="POST">
                       {{ .CsrfField }}
                       <div class="form-group">
-                        <label for="email">Email address</label>
-                        <div class="input-group">
-                          <input value="{{ .Subscription.Email }}" disabled inputmode="email" type="text" maxlength="100" class="form-control" autocomplete="email" id="email" name="email" />
-                          <div class="input-group-append">
-                            <button id="email-edit-button" onclick="toggleButton(this)" type="button" class="btn btn-outline-primary">Edit</button>
-                          </div>
-                        </div>
+                        <label for="password">Confirm Current Password</label>
+                        <input required type="password" minlength="5" maxlength="256" class="form-control" autocomplete="password" id="password" name="password" />
+                      </div>
+                      <div class="form-group">
+                        <label for="email">New Email address</label>
```

Comment:
**minor:** It was written like this before too but should the first letter of `address` be capitalized to be consistent with the rest of the page?

Reply:
fixed